raiden/tests/utils/transfer.py

```diff
@@ -63,12 +63,12 @@ def direct_transfer(initiator_app, target_app, asset, amount):
     initiator_app.raiden.api.transfer(
         asset,
         amount,
-        1,  # TODO: fill in identifier
-        target_app.raiden.address
+        target_app.raiden.address,
+        identifier,
```

Comment:
compared to https://github.com/raiden-network/raiden/pull/147/files#diff-6e2cae78b3ca096e55f149bbba21ad04R84

is `identifier==None` a valid for `raiden.api.transfer`? Or rather: why not handling all the default identifier logic on the same level?